### PR TITLE
ui refactoring

### PR DESCRIPTION
**Pull request by petrjasek**
putting together initial structure for the ui refactoring. also trying to be a bit more consistent regarding styling with superdesk core, so making some eslint config changes as well.

trying to reuse existing store for now, to get some test data, not sure yet how to approach that part anyhow.

- [x] new route for refactored ui - `/planning2` - not available via menu
- [x] defining top level components for layout (searchbar, filtersbar, main, preview, filters)
- [x] preview panel components
- [x] filters panel components
- [x] editor component
...

### CODE REVIEW

**Comment by petrjasek** [comment]
@nrvikas @mdhaman @nrvikas does it make sense?

File: client/controllers/PlanningController.js [hunk 1]
```diff
@@ -53,10 +53,12 @@ export function PlanningController(
                 store.dispatch(actions.resetStore())
             })
 
+            const App = $scope.vm.app || PlanningApp;
```
Comment (mdhaman) [inline client/controllers/PlanningController.js]: Why do we need to have `PlanningApp` component on scope?

**Comment by mdhaman** [comment]
@petrjasek good that styling is consistent with `superdesk-core`. Not sure why it there was in first place.
Rest is all fine.

File: client/controllers/PlanningController.js (hunk 1 above)
Comment (petrjasek) [reply client/controllers/PlanningController.js]: this is just temporary so I can use same controller for both old and new app

**Comment by petrjasek** [comment]
I will leave it now, added missing layout components for filters panel, now it can be used to implement the functionality

**Pull request merged**